src/components/common/PageCard.tsx

```diff
@@ -87,15 +80,8 @@ export default function PageCard(props: PageCardProps): JSX.Element {
     }
   };
 
-  const goToPage = (event?: React.MouseEvent) => {
-    if (linkStyle === 'button') {
-      return;
-    }
-    history.push({ pathname: link });
-  };
-
   return (
-    <Box className={`${classes.container} ${isMobile ? '' : 'min-height'}`} onClick={goToPage}>
+    <Box className={`${classes.container} ${isMobile ? '' : 'min-height'}`}>
```

Comment:
interesting.. i should let Ming know.. current funder portal has the entire card as clickable but similar design as here (bottom link but in a card)...